### ISSUE TITLE
really slow

### ISSUE THREAD

**Issue by driff** [blackbeared/fusion#1]
the components are really slow, the datepicker's animation is laggy and the the dialog takes some time to show

**Comment by sandipsavaliya112** [comment]
Yeah, i am currently working on Datepicker as well as SharedDialog. @driff  Well thank you very much to take time and review it.
